File: Php/app/Urls/PlayerAddConfirmationUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class PlayerAddConfirmationUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::playerAddConfirmation, $homegame);
		}

	}

}

File: Web/Models/UrlModels/PlayerAddConfirmationUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{
    public class PlayerAddConfirmationUrlModel : HomegameUrlModel{

		public PlayerAddConfirmationUrlModel(Homegame homegame)
            : base(RouteFormats.PlayerAddConfirmation, homegame)
        {
		}

	}

}